packages/edit-widgets/src/blocks/widget-area/editor.scss

```diff
@@ -11,6 +11,15 @@
 	}
 }
 
-.wp-block-widget-area > .components-panel__body > .block-editor-inner-blocks {
+// Add some spacing above the inner blocks so that the block toolbar doesn't
+// overlap the panel header.
+.wp-block-widget-area > .components-panel__body > div > .block-editor-inner-blocks {
```

Comment:
I've also fixed this selector, which was broken, and added some comments for both.